repos/spack_repo/builtin/packages/cabana/package.py

```diff
@@ -93,8 +93,9 @@ class Cabana(CMakePackage, CudaPackage, ROCmPackage):
     # Dependencies for subpackages
     depends_on("all-library", when="@0.5.0:+all")
     depends_on("arborx", when="+arborx")
-    depends_on("hypre-cmake@2.22.0:", when="@0.4.0:+hypre")
-    depends_on("hypre-cmake@2.22.1:", when="@0.5.0:+hypre")
+    depends_on("hypre-cmake@2.22.0:", when="@0.4.0 +hypre")
+    depends_on("hypre-cmake@2.22.1:", when="@0.5.0:0.7.0 +hypre")
+    depends_on("hypre@3.0.0:", when="@0.8.0:+hypre")
```

Comment:
To clarify, this covers `master` and the planned update in Cabana? Seems to be the case if this is building